runtime/Cpp/runtime/src/InputMismatchException.cpp

```diff
@@ -13,3 +13,7 @@ InputMismatchException::InputMismatchException(Parser *recognizer)
   : RecognitionException(recognizer, recognizer->getInputStream(), recognizer->getContext(),
                          recognizer->getCurrentToken()) {
 }
+
+InputMismatchException::~InputMismatchException()
```

Comment:
As in the previous two comments. Happy make a choice between empty destructors and "= default"